### PR TITLE
Solo abductor is now possible if only one player signs up for abductor

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

If only one ghost chooses to say yes to abductor, they get to be a solo abductor, instead of there being no abductors at all.

## Why It's Good For The Game

Because trashing a ruleset because only one person instead of two said yes is dumb.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
tweak: Solo abductors can now be made if only one ghost signs up to be an abductor.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/abductor/abductor.dm [hunk 1]
```diff
@@ -34,10 +34,6 @@
 	name = "Abductor Solo"
 	outfit = /datum/outfit/abductor/scientist/onemanteam
 
-/datum/antagonist/abductor/scientist/onemanteam
```
Comment (Absolucy) [inline code/modules/antagonists/abductor/abductor.dm:37]: no clue why this was duplicated

**Comment by PowerfulBacon** [comment]
Abductor is a team operator and doesn't work as well with solo abductor. Solo abductor shouldnt exist

**Comment by Absolucy** [comment]
I mean, I've played solo abductor before (due to my agent having to leave and no ghosts being willing to take the agent's body) and it worked out fairly well.

**Comment by imsynthex** [comment]
Ultimately 2 abductors is worse than 1 since it's pretty clunky to be an agent and try to communicate with the scientist to teleport himself down and you up at the right times (this serves as a good balancing measure), all of that is eliminated by just playing solo since there's 2 less steps to worry about on each abduction. So I don't think this PR is a good idea because having solo abductors be a more common occurrence would mean you have more of those rounds where 2/3rds of the station is just brainwashed and puking out plasma

**Comment by Isy232** [comment]
I'd argue for a system that refunds the midround, let it roll something one person can spawn as instead of just spawning nothing.
Alternatively re-roll the abductor poll 5 or 10 minutes later until two people say yes.

Abductors are an overpowered antagonist as they are, if they cooperate well its basically impossible to catch them. A solo abductor having all the tools of both with no need to cooperate would be impossible to beat.

**Comment by itsmeow** [comment]
> I'd argue for a system that refunds the midround, let it roll something one person can spawn as instead of just spawning nothing. Alternatively re-roll the abductor poll 5 or 10 minutes later until two people say yes.

This is how it works. If not enough players sign up for a ruleset it is refunded

**Comment by Isy232** [comment]
> > I'd argue for a system that refunds the midround, let it roll something one person can spawn as instead of just spawning nothing. Alternatively re-roll the abductor poll 5 or 10 minutes later until two people say yes.
> 
> This is how it works. If not enough players sign up for a ruleset it is refunded

How quickly? I seem to remember abductors failing to spawn and no further midrounds for ~20 minutes,

My thought is it seems that maybe it refunds the threat points, however it doesn't refund the rolled spawn an antagonist.

I might not completely be right in this, however from what I've seen even if the threat is refunded, there often isn't another spawn for a very long time.

**Comment by Isy232** [comment]
> No, it doesn't trigger another midround immediately. For most midrounds only 1 sign up is required so a failure means no one is gonna play midrounds

Ah, my thought would be a delay of ~5-10 minutes, alternatively immediately re-roll a solo midround.

**Comment by itsmeow** [comment]
:key:

Abductor is deliberately designed with two players in mind. Solo abductor is not supposed to even work really.

If you want to change dynamic so that a midround injection is triggered within the next 5-ish minutes, that might be a good idea, but it's not a particularly pressing matter.

**Pull request closed without merging**